src/video/kmsdrm/SDL_kmsdrmvideo.c

```diff
@@ -79,16 +81,17 @@ get_driindex(void)
     int devindex = -1;
     DIR *folder = opendir(device);
     if (!folder) {
-        SDL_SetError("Failed to open directory '%s'.\n", device);
+        SDL_SetError("Failed to open directory '%s'", device);
         return -ENOENT;
     }
 
-    memcpy(device + KMSDRM_DRI_PATHSIZE, KMSDRM_DRI_DEVNAME,
-           KMSDRM_DRI_DEVNAMESIZE);
+    strncpy(device + KMSDRM_DRI_PATHSIZE, KMSDRM_DRI_DEVNAME,
```

Comment:
Please use `SDL_strlcpy()`
It's not super important here, but on Windows there's no C runtime dependency so we use all SDL functions internally.

Reply:
Done